bammetrics/src/main/resources/nl/lumc/sasc/biopet/pipelines/bammetrics/rnaHistogram.ssp

```diff
@@ -62,27 +61,25 @@
         #end
     </tr></thead>
     <tbody>
-        #for (sample <- samples.toList.sorted)
+        #for (sample <- samples.sortBy(_.name))
```

Comment:
I agree there is some code duplication. To fix this all templates should be rearranged. This is a bit related to BIOPET-628. Because this change is to large to do in the current issue I will make a new issue for this to look into.